entrypoint.sh

```diff
@@ -13,26 +13,35 @@ function bump_version(){
 }
 
 function create_release_branch() {
-    read -p 'Bump type (possible values: patch or minor): ' release_type
+    release_type=$1
+
     if [ "$release_type" != "patch" ] && [ "$release_type" != "minor" ]; then
-        echo "Wrong Bump type. It should be minor or patch"
+        echo "Wrong Bump type. It should be minor or patch. Received: $release_type"
         exit 1
     fi
-    read -p 'from commit (default: HEAD): ' commit
-    commit="${commit:=HEAD}"
 
-    git pull -q
+    commit=$2
+
+    commit="${commit:=HEAD}"  # set commit to HEAD if commit variable is null
+    echo "commit: $commit"
+
+    git pull --quiet
 
     #get the new version
     new_version=$(bump_version dry-run)
-    echo "New version: $new_version"
+    echo "New version (bump_version dry-run): $new_version"
 
-    #create a branch from the specified sha and and commit the version bump
+    #create a branch from the specified sha and commit the version bump
     git checkout -B release/$new_version $commit
     new_version=$(bump_version commit)
-    if git push -f -q; then
+    echo "New version (bump_version commit): $new_version"
+
+    if git push --force --quiet --set-upstream origin release/$new_version; then
```

Comment:
I am not sure there is a good reason to push force after all, you can remove it.